packages/gatsby/src/schema/__tests__/queries.js

```diff
@@ -1646,7 +1764,7 @@ describe(`Query schema`, () => {
     /**
      * queries are read from file and parsed with babel
      */
-    it.only(`escape sequences work when correctly escaped`, async () => {
```

Comment:
😬 Good catch!

Reply:
I'm happy to say the other tests were all passing when I removed that!